lib/routes/content.js

```diff
@@ -85,38 +84,39 @@ const routes = application => {
      * @param {object} next - Express callback
     */
     app.get(`/${config.content.base}/:type/${config.content.actions.add}`, (req, res, next) => {
-      const type = utils.singleItem('id', req.params.type.toLowerCase(), types);
-      const workflow = workflows.workflow(type, allflows, config, req);
-      const errors = _.get(req.session, `form.content.add[${req.params.type.toLowerCase()}].errors`, {});
-      const values = _.get(req.session, `form.content.add[${req.params.type.toLowerCase()}].content`, {});
+      utils.fill(types, req.params.type.toLowerCase(), references, database).then(type => {
```

Comment:
I missed the reason of change here - what changed from populating reference in content-types/merge?
My assumption is it's to have the most up-to-date references to select from